scoring/management/commands/import_plan_scores.py

```diff
@@ -186,8 +186,8 @@ def import_overall_scores(self):
                 council=council, year=self.YEAR
             )
 
-            plan_score.total = round(row["raw_total"] * 100, 1)
-            plan_score.weighted_total = round(row["weighted_total"] * 100, 1)
+            plan_score.total = round(row["raw_total"] * 100, 3)
+            plan_score.weighted_total = round(row["weighted_total"] * 100, 2)
```

Comment:
Any reason these are different?

Reply:
The reason is I did this at the end of the day and was tired :| 

So yes, will make both of them 3.